CoreGraphUtilities/src/au/gov/asd/tac/constellation/graph/utilities/hashmod/HashmodAction.java

```diff
@@ -120,14 +126,25 @@ private static void run(final GraphWriteMethods wg, final PluginInteraction inte
                     }
                 }
 
-                final int newAttribute = wg.addAttribute(GraphElementType.VERTEX, newAttributeType, nextAttr, nextAttr, "", null);
-                if (newAttribute != Graph.NOT_FOUND) {
-                    attributeValues[attrCount] = newAttribute;
-                    csvValues[attrCount] = i;
-                    attrCount++;
+                if (createAttributes) {
+                    final int newAttribute = wg.addAttribute(GraphElementType.VERTEX, newAttributeType, nextAttr, nextAttr, "", null);
+                    if (newAttribute != Graph.NOT_FOUND) {
+                        attributeValues[attrCount] = newAttribute;
+                        csvValues[attrCount] = i;
+                        attrCount++;
+                    }
+                }
+
+                if (createTransactions && !hashmod.getTransactionAttribute(nextAttr).equalsIgnoreCase("")) {
```

Comment:
I would change `!hashmod.getTransactionAttribute(nextAttr).equalsIgnoreCase("")` to `StringUtils.isNotBlank(hashmod.getTransactionAttribute(nextAttr))`

Reply:
Made the change!